scripts/uiupdate.sh

```diff
@@ -8,12 +8,22 @@ set -e
 if [ -z "$UI_VERSION_FILE" ]; then
 	echo "Must set UI_VERSION_FILE"; exit 1
 fi
+
 if [ -z "$UI_COMMITISH" ]; then
 	echo "Must set UI_COMMITISH"; exit 1
 fi
 
+UI_EDITION=$(make --no-print-directory edition)
+
+if [ "$UI_EDITION" == "oss" ]; then
```

Comment:
None blocking question and just wondering about it:
- The logic of finding out the version is within the main make file. Which is good.
- Then I see `uiupdate.sh` and `build-ui` are using (and repeating) same logic to determinate: `UI_EDITION`, `UI_REPO` and `UI_VERSION`.

Do you think in the future we could move that logic to the make file? so we apply this logic once, set those ENV variables and when using `build-ui` or `uiupdate` those are already set.

Do you think that could be douable & beneficial for the future?

Reply:
I don't know makefiles that well but because the `edition` comes from the target, I'm not exactly sure how I would set it like we do for our static variables. I could do it in each target before the script but that's basically the same thing.